py/picca/data.py

```diff
@@ -232,6 +235,19 @@ def add_dla(self,zabs,nhi,mask=None):
             self.diff = self.diff[w]
             self.reso = self.reso[w]
 
+    def add_absorber(self,lambda_absorber):
+        if not hasattr(self,'ll'):
+            return
+
+        w = sp.ones(self.ll.size).astype(bool)
+        w = w & (sp.fabs(1e4*(self.ll-sp.log10(lambda_absorber)))>forest.absorber_mask)
```

Comment:
`w &=` is more beautiful.
I don't understand the `1e4*`, is it because of the size of the pixel? It is not very robust.

Reply:
The pixels size is hardcoded to SDSS, which isn't great, but note that the factor 1e-4 is already hardcoded elsewhere in do_deltas so I decided not to change that in this PR

Reply:
but you can simply use `forest.dll` no?

Reply:
forest.dll is rescaled by args.rebin though. I want spectral pixel (i.e. rebin=1) always.

Reply:
why?

Reply:
Because the absorber mask is then always defined in this way, independent of other choices. People talk about masking pixels at the absorber wavelength and I want to avoid any confusion what "pixel" means.